certified-connectors/Alkymi/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1301 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "1.0.0",
+    "title": "Alkymi",
+    "description": "Programmatic interface for the Alkymi Data Inbox",
```

Comment:
@tori-alkymi Please provide a detailed description that everyone can understand and it should describe concisely the main purpose and value offered by your connector.

Docs - [Write a description for your connector](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#write-a-description-for-your-connector)

Reply:
@tori-alkymi Can you please let us know why this comment was resolved?

Reply:
@tori-alkymi Please provide a detailed description that everyone can understand and it should describe concisely the main purpose and value offered by your connector.